taxonomies/labels.txt

```diff
@@ -3727,6 +3727,20 @@ label_categories:en: en:Geographic label
 origins:en: en:sweden
 web:sv: https://fransverige.se/
 
+< sv:Från Sverige
+xx: Kött från Sverige
```

Comment:
Could you also add the en: translations?

Reply:
I mean, the translation would be “Meat from Sweden”, but there’s no “Meat from Sweden” label (that I know of), so even in English it should be displayed as “Kött från Sverige”, no?

Even in the English blurb on https://fransverige.se/other-languages/ it gives the labels using the Swedish names (emphasis mine):
> When you choose **‘Från Sverige’** (From Sweden) labelled raw materials, food, and plants, you get products loaded with added Swedish value. […] Both small and large companies label their products with **‘Från Sverige’** (From Sweden), **‘Kött från Sverige’** (Meat from Sweden), and **‘Mjölk från Sverige’** (Milk from Sweden) – making it easy to choose Swedish-produced goods. […]

Reply:
That's what we do for many other labels that have logos, like "Produit en Bretagne", we make "en: Produced in Brittany" the canonical entry name. We usually do this for all labels that have a name like "Made in [country]", or "[ingredient] from [country]", things like "British Eggs" etc.  It makes more sense for people who don't know the language.